telemetry/vscode/tst/generatorTest.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { generate } from '../src/generate'
+import { tmpdir } from 'os'
+import { readFileSync } from 'fs-extra'
+
+describe('Generator', () => {
+    let tempDir: string
+    beforeAll(() => {
+        tempDir = tmpdir()
+    })
+
+    test('Generate fails when validation fails', async () => {
+        try {
+            await testGenerator(['resources/invalidInput.json'], '/invalid/file/path' )
+        } catch (e) {
+            expect(e).not.toBeNull
+        }
```

Comment:
if the generator doesn't throw then the test passes. 

```suggestion
            return
        }
        assert.fail()
```

Reply:
Forgot this, added